bumble/transport/usb.py

```diff
@@ -268,22 +271,32 @@ async def close(self):
         found = None
         if ':' in spec:
             vendor_id, product_id = spec.split(':')
-            found = context.getByVendorIDAndProductID(int(vendor_id, 16), int(product_id, 16), skip_on_error=True)
+            if '/' in product_id:
+                product_id, serial_number = product_id.split('/')
+                for device in context.getDeviceIterator(skip_on_error=True):
+                    if (
+                        device.getVendorID() == int(vendor_id, 16) and
+                        device.getProductID() == int(product_id, 16) and
+                        device.getSerialNumber() == serial_number
+                    ):
+                        found = device
+                        break
+                device.close()
```

Comment:
You want to close the `device` from the iterator right? I think this line should be indented one more level to be inside the `for` loop.

Reply:
yes, indeed.

Reply:
now fixed in latest commit